washee_web/booking/tests/conftest.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from datetime import datetime
+
+from booking.models import Booking
+from account.models import User
+from location.models import Service, MachineModel, Machine, Location
+
+@pytest.fixture
+def create_booking_func():
+    """
+        Creates a booking. 
+        TODO: Fixtures should be made in their respective apps, instead of here.
+        TODO: All these extra fixtures should follow this interface (many "model_attribute" and a "save=False" option)
+    """
+    def _create_booking(
```

Comment:
This fixture is so big that it should have its own tests.
I believe the best idea would be to put these kind of fixtures in "washee_web/washee_web/tests" and then make tests that verify that the fixtures actually do as expected.

Right now, as long as the interface doesn't change, this should be more than fine for the purposes. The tests are at least incredibly readable!